src/client/resources/discovery.ts

```diff
@@ -0,0 +1,26 @@
+import { ClusterDiscoveryResource, IDiscoveryParametrs } from './common';
+
+export class NamespaceDiscovery extends ClusterDiscoveryResource {
+  private _discoveryParameters: IDiscoveryParametrs;
+
+  constructor(cluster: string, { wait = -1, offset = -1, limit = -1 } = {}) {
```

Comment:
RC: See magic number comment, should be a const defined somewhere and used here to default it.